web/templates/settings/show.html.eex

```diff
@@ -39,13 +39,13 @@
           Send me daily digest emails about what's new on Constable.
           </label>
         </div>
-      </div>
-
-      <footer>
-        <%# <%= link "Logout of Constable", to: session_path(@conn, :delete), method: :delete %1> %>
         <%= submit gettext("Save") %>
-      </footer>
-    <% end %>
+      <% end %>
+    </div>
+
+    <footer>
+      <%= link "Logout of Constable", to: session_path(@conn, :delete), method: :delete, class: "button button-secondary" %>
```

Comment:
I know the text was already there, but I think we may want `Log out` since it's a verb as opposed to when its used as a noun (e.g. "Logout page").